webpack/assets/javascripts/react_app/components/common/table/TableSelection.stories.mdx

```diff
@@ -0,0 +1,84 @@
+import { Meta } from '@theforeman/stories';
+import { Icon } from 'patternfly-react';
+
+<Meta
+  title="Page chunks|Table"
+  parameters={{
+    storyWeight: 90,
+  }}
+/>
+
+# Table With Select boxes
+
+All needed function and components are located at: `/react_app/components/common/table/index.js`  
+To add select boxes to your foreman table please follow this steps:
+
+## Connect The Reducer
+
+Connect the `selectionReducer` from the table folder to your component
+
+<Icon type="pf" name="info" />
+{' '}Tip: You can connect a few reducers to one component using Redux's
+<strong> combineReducers</strong>
+
+### Data provided by the reducer
+
+**allRowsSelected**: a boolean that describes if all the rows available are selected  
+**selectedRows**: an array of the selected ids  
+**showSelectAll**: a boolean that indicates should the select all option be available?  
+This is true after a user is selection the whole page. Use this to show a select all button or message box.
+
+## Connect The Actions
+
+Create a `selectionController` using the `getSelectionController` helper
+
+```js
+getSelectionController({
```

Comment:
When hooking up multiple tables, the dispatched actions will have same types for different tables if I am reading this correctly. I am not sure if redux will be able to update only the desired part of the store that way...

Reply:
Thanks for the feedback, maybe there can be general reducers and selectors like in the API

Reply:
Each action should have a table id to work now